hopsworks-common/src/main/java/io/hops/hopsworks/common/exception/ThrowableMapper.java

```diff
@@ -87,8 +87,8 @@ public Response toResponse(Throwable ex) {
   }
   
   public Response handleIllegalArgumentException(IllegalArgumentException ex) {
-    return handleRESTException(new GenericException(RESTCodes.GenericErrorCode.ILLEGAL_ARGUMENT, Level.FINE, null,
-      ex.getMessage(), ex));
+    return handleRESTException(new GenericException(RESTCodes.GenericErrorCode.ILLEGAL_ARGUMENT, Level.FINE,
+        ex.getMessage(), ex.getMessage(), ex));
```

Comment:
```suggestion
        ex.getMessage(), null, ex));
```
@SirOibaf why send the message both in usrMsg and devMsg. It should be set only once, it's the client's responsibility to parse the fields it wants. Also I've changed this logging [here](https://github.com/logicalclocks/hopsworks/pull/47)

Reply:
Ok. I modified it because I have some tests who trigger these exceptions and I wanted to check the error message. When I changed it, I didn't know when you were going to merge your pr